### PR TITLE
Show severity in Github Workflow command message

### PR DESCRIPTION
**Pull request by JiriSko**
Fixes: #1054

### CODE REVIEW

File: lib/ansiblelint/formatters/__init__.py [hunk 1]
```diff
@@ -123,10 +123,11 @@ def format(self, match: "MatchError", colored: bool = False) -> str:
         file_path = self._format_path(match.filename or "")
         line_num = match.linenumber
         rule_id = match.rule.id
+        severity = match.rule.severity
         violation_details = match.message
         return (
             f"::{level} file={file_path},line={line_num}"
-            f"::[E{rule_id}] {violation_details}"
+            f"::[E{rule_id}] [{severity}] {violation_details}"
```
Comment (ssbarnea) [inline lib/ansiblelint/formatters/__init__.py:130]: Based on how the format looks, what you want is to add `,severity={}` on the first line and avoid polluting the message itself.

**Review by rafaelfolco — COMMENTED** [review]
This worked for me only when I specify
ansible-lint --parseable-severity roles/test.yml 

When I use -p (pep8 format), the severity info is gone.
Is this expected ?

**Review by ssbarnea — CHANGES_REQUESTED** [review]
Please convert it to draft until you pass all tests.

**Comment by ssbarnea** [comment]
Code looks ok now but before merging it, do you have a proof it does work? My guess is that github should just ignore the message but it could also prevent it from rendering them. 

Can you please made a manual test of producing such an annotation and confirm that it works?

The second question would, what if github does not display severity at all in the UI. How is this feature going to help the user?

I am quite curious about you use-case around priority as I never seen it more than something put inside the rule documentation.

**Comment by ssbarnea** [comment]
@JiriSko If you cannot produce an example that proves that github actions would not choke when receiving an additional parameter and that it is possible to view the severity on the annotations produced I will have to close this PR.

Even a screenshot is enough. You should be able to test this with a simple print that produces an annotation that is using this format.

The only reason I am asking this is because I want to prevent a potential break in current code by merging this feature without confirmation that it works.

**Comment by JiriSko** [comment]
Sorry it took so long. Here's test with old and new annotation format:
https://github.com/codeacio/ansible-lint/actions/runs/289711321

**Comment by ssbarnea** [comment]
@jimi-c No worries. Now we know that github accepts extra data in severity. Still, my impression is that the severity will not be visible to the user. How is user going to make use of it?

**Comment by JiriSko** [comment]
It's part of my CI to categorize issues to errors and warnings. Isn't visible directly in Github. I'm using it in previous version of ansible-lint.

**Review by ssbarnea — APPROVED** [review]

**Pull request merged**